nixos/modules/hardware/gkraken.nix

```diff
@@ -3,16 +3,17 @@
 with lib;
 
 let
+  inherit (lib)
+    mdDoc mkEnableOption mkIf;
```

Comment:
Each of those functions is used once. The file is not even 20 lines long. Both the inherit and with should be dropped and the three occurrences of lib should just be written out.

Reply:
Being redundant, my primary concern here is to make the code compile without `with`.

After this, it will be easy to prefix lib.

OTOH I don't believe this is bad code. It is similar to other programming languages like C++ namespaces and Python lib imports.